packages/fxa-auth-server/lib/payments/iap/apple-app-store/types/api-client.ts

```diff
@@ -0,0 +1,186 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2021 August Heegaard
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+// Below types are copied from 'app-store-server-api/dist/types/Models'
+// as only the enum types (a minority) are available in 'app-store-server-api/dist/cjs/',
+// and TS in ./node_modules are not compiled into JS along with the fxa source code.
```

Comment:
@bbangert : I couldn't get rid of this file after all; see code comments.

Reply:
Ben and I discussed this in Slack; I filed https://mozilla-hub.atlassian.net/browse/FXA-4920.

Per Ben:
> My main worry about copying the types in, is it makes our code brittle. A dependabot push to bump our use of his library can easily break things if he changes the types at all if they're not a public API from the library.